### PR TITLE
Implement retry logic for failed writes in influx_stress

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

**Comment by jackzampolin** [comment]
@mjdesa

File: stress/basic.go [hunk 1]
```diff
@@ -245,7 +245,26 @@ type BasicClient struct {
...
 	Format        string   `toml:"format"`
-	addrId        int
+
+	addrId   int
+	r        chan<- response
+	interval time.Duration
+}
+
+func (c *BasicClient) retry(b []byte, backoff time.Duration) {
+	bo := backoff + time.Duration(500*time.Millisecond)
```
Comment (otoolep) [inline stress/basic.go]: Why add 500 milliseconds?

Comment (jackzampolin) [inline stress/basic.go]: To increase the backoff for each additional retry.  An exponential backoff could look like:
```
// Function is invoked with the arg time.Duration(1)
if backoff != time.Duration(1) {
  bo := backoff * time.Duration(2*time.Millisecond)
} else {
  bo := time.Duration(500*time.Millisecond)
}
```

Comment (otoolep) [inline stress/basic.go]: So you just want a linear increase?

Comment (jackzampolin) [inline stress/basic.go]: We tested with the linear increase and it worked well.  Did not see the need for the exponential increase.

Comment (otoolep) [inline stress/basic.go]: OK, great, as long as you know what you are doing. 

That said, this value should be defined as a constant at the top of the file, and not exist as a magic number.

**Comment by mjdesa** [comment]
+1 on green

**Pull request merged**